pySDC/implementations/problem_classes/AllenCahn_1D_FD.py

```diff
@@ -735,7 +717,7 @@ def __init__(
         stop_at_nan=True,
     ):
         super().__init__(nvars, dw, eps, newton_maxiter, newton_tol, interval, radius, stop_at_nan)
-        self.A -= sp.eye(self.init) * 0.0 / self.eps**2
+        self.A -= sp.eye(self.nvars) * 0.0 / self.eps**2
```

Comment:
Does this line do anything? It just subtracts zero. At least with your fix this does not raise an error, but still. Is there a reason for this line @pancetta?

Reply:
There was, but not anymore. I don't know why this is still in there. Can be removed!